### PR TITLE
Standardize e2e naming 2

### PR DESCRIPTION
**Pull request by Hugo-Hontans**

### CODE REVIEW

File: apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts [hunk 1]
```diff
@@ -1,56 +1,61 @@
...
+  ////////////////////
+
   public openProfileMenu() {
     cy.get('[page-id=navbar]').get('button[test-id=profile-avatar]').click();
   }
 
+  public clickProfile() {
+    cy.get('button[test-id=profile]').click();
+    return new ProfileEditablePage();
+  }
```
Comment (ioaNikas) [inline apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts]: Is this working without calling `openProfileMenu()` function  to open the menu before clicking on profile ?

File: apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts [hunk 2]
```diff
@@ -1,56 +1,61 @@
...
+  public clickContextMenuInformation(): DeliveryInformationsEditablePage {
+    cy.get('[page-id=navbar] a').contains('information').click();
+    return new DeliveryInformationsEditablePage();
   }
 
   public clickContextMenuTemplatesCreate() {
-    cy.get('[page-id=navbar]').contains('templates').click();
+    cy.get('[page-id=navbar] a').contains('templates').click();
     return new TemplateCreatePage();
   }
```
Comment (ioaNikas) [inline apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts]: We should either specify the return type of all the functions or never specify it. The choice is your !

File: libs/material/src/lib/delivery/pages/delivery-informations-editable/delivery-informations-editable.component.ts [hunk 3]
```diff
@@ -12,6 +12,8 @@ import { createInformationsFormGroup } from '../../forms/informations.form';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DeliveryInformationsEditableComponent implements OnInit {
+  @HostBinding('attr.page-id') pageId = 'delivery-informations-editable';
+
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/pages/delivery-informations-editable/delivery-informations-editable.component.ts]: We should rename the entire component to `DeliveryInformationEditableComponent` (without 's'), and do the same for the selector, HostBinding etc.

File: apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts (hunk 2 above)
Comment (Hugo-Hontans) [reply apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts]: For me it's not required but if the developer wants add it, he is free. It's just add a specification.

File: libs/material/src/lib/delivery/pages/delivery-informations-editable/delivery-informations-editable.component.ts (hunk 3 above)
Comment (Hugo-Hontans) [reply libs/material/src/lib/delivery/pages/delivery-informations-editable/delivery-informations-editable.component.ts]: Yes but it's really not the subject of my PR but I can do it in next

File: apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts (hunk 1 above)
Comment (Hugo-Hontans) [reply apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts]: No I can standardize it, you right

**Pull request merged**